README.md

```diff
@@ -1625,7 +1693,7 @@ class CustomExporter(Exporter):
 
 
 # Register the new custom class into factory
-exporter_factory.register_exporter("custom", CustomExporter)
+exporter_factory.register_exporter("custom_exporter", CustomExporter)
```

Comment:
`custom` export_format is used in this feature, so I changed `CustomExporter` code examples.